src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -91,6 +94,20 @@ case class Submission(
   def idFields = Seq("submissionId")
 }
 
+case class SubmissionStatusResponse(
```

Comment:
a new response class that is passed back instead of Submission so that submitter still looks like an email address to the caller